newIDE/app/src/Utils/GDevelopServices/Authentication.js

```diff
@@ -125,6 +134,10 @@ export default class Authentication {
     });
   }
 
+  setLoginProvider = (loginProvider: LoginProvider) => {
+    this.loginProvider = loginProvider;
```

Comment:
this could be set when auth is created, in the `src/index.js` instead of having the "chance" of it not being defined

Reply:
There is no notion of Local/Browser/Mobile in `src/index.js` so I didn't want to bring it there